package.json

```diff
@@ -40,7 +40,7 @@
     "lodash": "^4.17.21",
     "lucide-react": "^0.368.0",
     "moment": "^2.29.4",
-    "protobufjs": "^7.2.5",
```

Comment:
@BDavid57 Why do we have protobufjs in dependencies?

Reply:
@alexberce I'm trying to solve the problem, I thought this will solve it